<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SshShellWidget</class>
 <widget class="QWidget" name="SshShellWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="siteCmbBox"/>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QLineEdit" name="cmdEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
